java/src/jmri/web/servlet/panel/LayoutPanelServlet.java

```diff
@@ -5,26 +5,14 @@
 import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
-import jmri.InstanceManager;
-import jmri.Sensor;
-import jmri.SensorManager;
-import jmri.Turnout;
-import jmri.TurnoutManager;
+import jmri.*;
```

Comment:
@geowar1 I've been told that the explicit imports are preferred in JMRI. Is this change intentional?

Reply:
> I've been told that the explicit imports are preferred in JMRI

There is no agreed-to recommendation, unfortunately.  Our [recommended practices page](https://www.jmri.org/help/en/html/doc/Technical/RP.shtml) is silent on this, referring to the [Java Code Conventions document](https://web.archive.org/web/20080703230238/http://java.sun.com/docs/codeconv/), which doesn't say anything specific and has examples of several types (pages 3, 4, and 19).

It comes up periodically.  Some people like every import to be explicit, and have reasons for that.  Others prefer generic imports, and have reasons for that.  Some people (like me) prefer generic if there's more than 3-4 imports from a single package, but are OK with individual explicit imports of a class or two.

Reply:
Ok, thanks, @bobjacobsen

Reply:
Just an FYI: I notice that if I wildcard the java.awt package it picks up java.awt.list which conflicts with List in java.util. So I avoid wildcarding the java.awt package.